gulpfile.js

```diff
@@ -107,11 +107,11 @@ gulp.task('default', () => gulp.parallel('sass', 'lint-sass'));
 gulp.task('build:sass', () => 
   gulp.src('./index.scss')
   .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
-  .pipe(gulp.dest('./dist'))
+  .pipe(gulp.dest('./build'))
 );
 
 gulp.task('clean:dist', () => 
-  del(['dist/**'])
+  del(['build/**'])
```

Comment:
Why the folder changed to build? In components it's dist as well

Reply:
I was just unsure if it might be used somewhere that's why I did a new name. Last build failed because .dist is in .gitignore.

Reply:
I guess dist is in gitignore by default^^ I would propose to remove it there and have it unified in all repos